randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -768,13 +768,15 @@ Extra - asset_id: collision_camera_011
       Trivial
   > Dock to Map Station Access (Upper)
       Any of the following:
-          Grapple Beam or Space Jump or Lay Cross Bomb
+          Grapple Beam or Space Jump
           Slide and Slide Jump (Beginner) and Use Spin Boost
           Flash Shift and Wall Jump (Beginner)
           Gravity Suit and Simple IBJ
           All of the following:
-              # This room allows for a bomb jump out of the water without a WBJ
-              Bomb and Morph Ball and Infinite Bomb Jump (Intermediate)
+              Water Bomb Jump (Intermediate)
+              Any of the following:
+                  Lay Cross Bomb
+                  Diagonal Bomb Jump (Intermediate) and Lay Normal Bomb
```

Comment:
Lay Cross Bomb should have IBJ (Beginner). This was a case of Lay Cross Bomb not being changed to IBJ way back.

Reply:
Yeah, this one is fine without IBJ I would think.